### ISSUE TITLE
"syntax error" when "using Escher"

### ISSUE THREAD

**Issue by eschnett** [shashi/Escher.jl#60]
I see this error:

```
julia> using Escher
WARNING: module Patchwork should explicitly import << from Base
WARNING: module DataStructures should explicitly import < from Base
WARNING: module DataStructures should explicitly import <= from Base
WARNING: module JSON should explicitly import colon from Base
WARNING: module JSON should explicitly import colon from Base
ERROR: LoadError: LoadError: syntax: malformed expression
 in include at /Users/eschnett/julia/lib/julia/sys.dylib
 in include_from_node1 at /Users/eschnett/julia/lib/julia/sys.dylib
 in include at /Users/eschnett/julia/lib/julia/sys.dylib
 in include_from_node1 at /Users/eschnett/julia/lib/julia/sys.dylib
 in require at /Users/eschnett/julia/lib/julia/sys.dylib
while loading /Users/eschnett/.julia/v0.4/Escher/src/basics/util.jl, in expression starting on line 62
while loading /Users/eschnett/.julia/v0.4/Escher/src/Escher.jl, in expression starting on line 22
```

This is with a fresh version of Julia/master.

**Comment by NOTtheMessiah** [comment]
While I got the warning messages from deprecated syntax, I did not get the LoadError. The dylib extension seems to be OS X exclusive, and I'm on Arch Linux. Can anyone with OS X reproduce this?

**Comment by rohitvarkey** [comment]
@eschnett `Pkg.checkout("Escher")` fixes it for me. 

@shashi Tag a new version to prevent this from happening?

**Comment by shashi** [comment]
Will tag once I fix some Polymer 1.0 migration issues. Don't want to break things for everyone.

**Comment by sjkelly** [comment]
This seems to be fixed.
